lua/weapons/weapon_ace_xm25/shared.lua

```diff
@@ -3,6 +3,7 @@ SWEP.Base = "weapon_ace_base"
 SWEP.Category = "ACE Weapons"
 SWEP.SubCategory = "Special"
 SWEP.Purpose = "Lob HE behind cover"
+SWEP.Instructions	= "Left click to shoot. Right click while sprinting to scope, Right click to lase"
```

Comment:
a bit weird naming for shift. I wonder, if there is a way to return the bind button user has for sprint

Reply:
The function returns when the user is pressing sprint